docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  api:
+    image: spritsail/fivem
+    container_name: container-name
```

Comment:
Perhaps `container_name: fivem`, or just leave it out altogether?
A name is a) personal preference and b) not required to start the container